lib/akamai_rspec/request.rb

```diff
@@ -69,7 +69,12 @@ def self.responsify(maybe_a_url)
       if maybe_a_url.is_a? RestClient::Response
         maybe_a_url
       else
-        RestClient.get(maybe_a_url, akamai_debug_headers)
+        begin
+          RestClient.get(maybe_a_url, akamai_debug_headers)
+        rescue RestClient::RequestFailed => exception
+          # Return the original request
+          exception.response
```

Comment:
What is the scenario for this? And is `exception.response` the original request?